zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/domain/rol/RolTest.kt

```diff
@@ -0,0 +1,99 @@
+package com.ritense.zakenapi.domain.rol
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.net.URI
+import kotlin.test.assertEquals
+
+internal class RolTest {
+
+    private val mapper = jacksonObjectMapper()
+
+    @Test
+    fun `should serialize natuurlijk persoon`() {
+        val rol = Rol(
+            URI("http://zaak.uri"),
+            URI("http://betrokkene.uri"),
+            BetrokkeneType.NATUURLIJK_PERSOON,
+            URI("http://role.type"),
+            "role description",
+            RolNatuurlijkPersoon(
+                inpBsn = "bsn"
+            )
+        )
+
+        val result = mapper.writeValueAsString(rol)
+
+        val expectation =  "{" +
```

Comment:
These strings can be made using """{...}""" to prevent all the escape characters

Reply:
Fixed